project/plugins.sbt

```diff
@@ -7,11 +7,13 @@
  * This file is part of the Apache Pekko project, which was derived from Akka.
  */
 
+dependencyOverrides += "org.scala-lang.modules" %% "scala-xml" % "2.2.0"
```

Comment:
@pjfanning Why is this here?

Reply:
Some of the old sbt plugins have scala-xml v1 dependencies. pekko-grpc-sbt-plugin now brings in scala-xml v2 transitively. So I need to force sbt to use scala-xml v2 to stop it failing to run due to mismatched scala-xml versions.